dts/riscv/it8xxx2_evb.dtsi

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (c) 2020 ITE Corporation. All Rights Reserved.
+ * Copyright (c) 2019-2020 Jyunlin Chen <jyunlin.chen@ite.com.tw>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+
+/ {
+	#address-cells = <1>;
+	#size-cells = <1>;
+
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+		cpu@0 {
+			clock-frequency = <0>;
+			compatible = "riscv-ite";
+			device_type = "cpu";
+			reg = <0>;
+		};
+	};
+
+	soc {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		ranges;
+
+		baseaddr: regbase@F00000 {
```

Comment:
What is this node?  it has no compatible property?

Reply:
It's all of the control register's base address, I think the compatible property should follow the soc? I will add this.

Reply:
Doesn't seem as if it used anywhere today in the code.

Reply:
This will be used in soc/riscv/riscv-ite/common/chip_chipregs.h .